src/modules/configurator/components/ConfigOptionsButton.tsx

```diff
@@ -8,7 +8,7 @@ export function ConfigOptionsButton({
   setIsOptionsDropdownOpen,
 }: Props) {
   return (
-    <button onClick={() => setIsOptionsDropdownOpen(!isOptionsDropdownOpen)}>
+    <button onClick={(e) => setIsOptionsDropdownOpen(!isOptionsDropdownOpen)}>
```

Comment:
Why did you put the `e` here when you are not using it?

Reply:
Removed, stayed there by mistake.